Applications/Spire/Source/TimeAndSales/TimeAndSalesTableView.cpp

```diff
@@ -13,6 +13,7 @@
 #include "Spire/Ui/TransitionWidget.hpp"
 
 using namespace Spire;
+using Columns = TimeAndSalesProperties::Columns;
```

Comment:
This should be renamed from ```Columns``` to ```Column``` and ```TimeAndSalesProperties::Columns``` should be renamed to ```TimeAndSalesProperties::Column```.

A type should be named based on a single instance. If an instance represents a single column, then the name should be ```Column```. If an instance represents multiple columns, only then would it make sense to name it ```Columns```.